models/product/src/product/product-draft/builder.spec.ts

```diff
@@ -180,17 +180,20 @@ describe('builder', () => {
       expect.objectContaining({
         name: expect.arrayContaining([
           expect.objectContaining({
-            __typename: 'LocalizedString',
+            locale: expect.any(String),
+            value: expect.any(String),
           }),
         ]),
         slug: expect.arrayContaining([
           expect.objectContaining({
-            __typename: 'LocalizedString',
+            locale: expect.any(String),
```

Comment:
`builder.spec.ts` for any draft where the `isGraphqlDraft` option is passed to the transformer needs to be updated manually, like so.